lua/lspconfig/server_configurations/astro.lua

```diff
@@ -13,17 +13,16 @@ return {
     root_dir = util.root_pattern('package.json', 'tsconfig.json', 'jsconfig.json', '.git'),
     init_options = {
       typescript = {
-        serverPath = '',
+        tsdk = '',
```

Comment:
if this field don't need set any value in there. can be don't set this field. just remove this line.

Reply:
Yup, almost forgot that it's what I do in my config.